packages/server/lib/util/ci_provider.js

```diff
@@ -154,12 +154,19 @@ const _providerCiParams = () => {
       'APPVEYOR_PULL_REQUEST_NUMBER',
       'APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH',
     ]),
+    // https://learn.microsoft.com/en-us/azure/devops/pipelines/build/variables
     azure: extract([
       'BUILD_BUILDID',
       'BUILD_BUILDNUMBER',
       'BUILD_CONTAINERID',
       'BUILD_REPOSITORY_URI',
+      'SYSTEM_JOBID',
+      'SYSTEM_STAGEATTEMPT',
+      'SYSTEM_PHASEATTEMPT',
+      'SYSTEM_JOBATTEMPT',
       'SYSTEM_PULLREQUEST_PULLREQUESTNUMBER',
+      'SYSTEM_PULLREQUEST_TARGETBRANCH',
+      'SYSTEM_PULLREQUEST_TARGETBRANCHNAME',
```

Comment:
We may want to collect SYSTEM_PLANID as well? 

```
System.PlanId | A string-based identifier for a single pipeline run.
```

Reply:
https://github.com/cypress-io/cypress/pull/29845/commits/e7ee14306e433353cb2e4cf93d010b53e4b74e6f